src/main/java/ru/yandex/practicum/filmorate/storage/film/FilmDbStorage.java

```diff
@@ -104,6 +105,12 @@ public Collection<Film> getPopularFilms(Integer count) {
         return setFilmGenres(films);
     }
 
+    @Override
+    public void deleteFilmById(Integer id) {
+        final String sql = "delete from films where id = ?";
+            jdbcTemplate.update(sql, id);
```

Comment:
```suggestion
            jdbcTemplate.update(sql, id);
```

 jdbcTemplate.update возвращает булево, количество измененных строк, если придет 0, то можно делать throw NotFound...
 Чтобы не делать лишний гет запрос перед удалением

Reply:
Ошибся, Integer возвращает